tests/test_tools.py

```diff
@@ -330,6 +330,16 @@ def test_get_conflicts(self, client, helpers):
         assert len(conflicts) == 1
         assert conflicts[0] == 'cmu.edu'
 
+        helpers.create_user('user@facebook.com', 'FirstA', 'Last')
+        user_profile = client.get_profile(email_or_id='user@facebook.com')
+
+        helpers.create_user('user2@fb.com', 'FirstB', 'Last')
+        user2_profile = client.get_profile(email_or_id='user2@fb.com')
+
+        conflicts = openreview.tools.get_conflicts([user_profile], user2_profile)
```

Comment:
can you add a test checking subdomains? something like cs.abdn.ac.uk vs cs.aberdeen.ac.uk 

FYI you don't need to create a profile in the database, creating the profile entity in memory is enough:

```        profile2 = openreview.Profile(
            id = 'Test_Conflict2',
            content = {
                'emails': ['user2@126.com'],
                'history': [{
                    'institution': {
                        'domain': 'user2@cmu.edu'
                    }
                }]
            }
        )
```

Please add a test where the duplicated domain is stored in the history section and not as an email. That should cover all the cases.